falcon/asgi/response.py

```diff
@@ -50,20 +50,22 @@ class Response(falcon.response.Response):
                 See also :ref:`media` for more information regarding media
                 handling.
 
-        body (str): String representing response content.
```

Comment:
It would be helpful to still include this in the docstring, yet note that it is a deprecated alias.